Migrate SQL docs (#521)
redpanda-data/cloud-docs: modules/reference/pages/sql/sql-functions/string-functions/length.adoc

= LENGTH

> :description: The LENGTH() function is used to find the length of a string, i.e., the number of characters in a given string.

Reviewer: ```suggestion
:description: The LENGTH() function is used to find the length of a string (for example, the number of characters in a given string).
```

:page-topic-type: reference

The `LENGTH()` function is used to find the length of a string, i.e., the number of characters in a given string. It accepts a string as a parameter. The syntax of the length function is as follows:

[source,sql]
----
LENGTH(string)
----

The input type is a string, and the return type is int, as it returns the number of characters.

*Special cases:*

* If a null value is passed in the function, i.e., `LENGTH(NULL)`, it returns `NULL`.
* If the parameter is an empty string `LENGTH(")`, it returns 0.
* If the parameter is a space character `LENGTH('')`, not empty or null, it returns 1 as it is not empty anymore.

== Examples

=== Basic `LENGTH()` function

> The following example uses the `LENGTH()` function to find out the length of a string text:

Reviewer: ```suggestion
This example uses the `LENGTH()` function to find out the length of a string text:
```
